ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/monitor-sessions.j2

```diff
@@ -0,0 +1,71 @@
+{# Leaf connected_endpoint monitor-sessions #}
+{% if monitor_sessions_data | length > 0 %}
+monitor_sessions:
+{%     for sname, group in monitor_sessions_data | groupby("name") %}
+{%         set group_roles = group | map(attribute='role', default='notdefined') | list %}
```

Comment:
we don't need this protective logic. It could easily happen that someone wants to feed in parts of the monitor config using `structured_config`, so we should allow for a "half" session (only source or only destination)